apps/client/src/app/pages/portfolio/fire/fire-page.html

```diff
@@ -163,6 +166,7 @@ <h4 class="align-items-center d-flex m-0">
         <gf-rules
           [hasPermissionToCreateOrder]="hasPermissionToCreateOrder"
           [rules]="feeRules"
+          (rulesUpdated)="onRulesUpdated($event)"
         />
       </div>
     </div>
```

Comment:
Could you add a new section here showing all inactive rules: `<span i18n>Inactive</span>`

Reply:
So not showing them in the report, and instead having them at the bottom in the Inactive section?

Reply:
Exactly. This is the starting point for further personalization options in the future.

Reply:
How would the Inactive section look like? Having each rules that are inactive listed one by one, or separated by their categories ?

Reply:
One by one to keep it more compact.